packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -3001,7 +3002,11 @@ describe('<Autocomplete />', () => {
       checkHighlightIs(getByRole('listbox'), 'one');
       setProps({ options: ['four', 'five'] });
       checkHighlightIs(getByRole('listbox'), 'four');
-      expect(handleHighlightChange).to.deep.equal([null, 'one', 'four']);
+
+      const expectedCallHistory =
+        reactMajor === 19 ? [null, 'one', 'one', 'four'] : [null, 'one', 'four'];
+
+      expect(handleHighlightChange).to.deep.equal(expectedCallHistory);
```

Comment:
I'm not sure why this one (or [the similar one for joy](https://github.com/mui/material-ui/pull/43155/files#diff-277e65a74ad823f227376e28e5cdcbd6c0552965a660cfb4f0c3720d67c0e365R2291)) happened, might be something related to the Strict mode changes, but the explanation on [the upgrade guide](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#strict-mode-improvements) is not very detailed.